Sources/Helpers/OASvgHelper.mm

```diff
@@ -15,6 +15,11 @@
 
 @implementation OASvgHelper
 
++ (BOOL)hasMapImageNamed:(NSString *)name
+{
+    return name && ([self mapImagePathFromSvgResource:[name hasPrefix:@"mx_"] ? name : [@"mx_" stringByAppendingString:name]]) != nil;
```

Comment:
We have also "c_mx_" images. Such common method should take it into consideration.